project/scripts/bisect.scala

```diff
@@ -238,6 +238,7 @@ class CommitBisect(validationScript: File, shouldFail: Boolean, bootstrapped: Bo
     val bisectRunScript = s"""
       |scalaVersion=$$(sbt "print ${scala3CompilerProject}/version" | tail -n1)
       |rm -r out
+      |export BISECTBUILD=yes
       |sbt "clean; ${scala3Project}/publishLocal"
```

Comment:
```suggestion
      |sbt "clean; set Compile / doc / skip := true; set Compile/ doc / sources := Nil; ${scala3Project}/publishLocal"
```
I think this single line change should be equivalent to all the other changes from this PR and it should speed up also builds of historical commits

Reply:
Not really, the global scope definition would not propagate to the projects, so these would have any effect. We could use `set every x` but this one does not support scopes, instead it would apply value to every task with given name. We could use internals to properly set correct values, but that's the overkill. 

However, we can use comination of 2 more specific commands to get the expected effect: 
* `set scaladoc/Compile/resourceGenerators := (`scala3-bootstrapped`/ Compile/resourceGenerators).value` - to remove custom resourceGenerators requiring compilation and linking scaladoc static resources using Scala.js
* `set every doc := new File("unused")` - overwrite all `doc` tasks to no-op (seems to be unharmfull, even though we use non-exisitng file)